app/celery/process_ses_receipts_tasks.py

```diff
@@ -103,7 +104,8 @@ def sns_callback_handler():
             response = requests.get(url, timeout=(3.05, 1))
             response.raise_for_status()
         except requests.RequestException as e:
-            current_app.logger.warning("Response: %s", response.text)
+            if response is not None:
+                current_app.logger.warning("Response: %s", response.text)
```

Comment:
We still need to log if the response is None, we just can't log the response.text
```suggestion
            current_app.logger.warning("RequestExcpetion - Response: %s", response.text if response is not None else e.response)
```